packages/pluggableWidgets/datagrid-web/src/components/__tests__/Header.spec.tsx

```diff
@@ -23,6 +23,7 @@ function mockHeaderProps(): HeaderProps<object> {
         sortable: false,
         setColumnOrder: jest.fn(),
         setDragOver: jest.fn(),
-        visibleColumns: []
+        visibleColumns: [],
+        setSortBy: jest.fn()
```

Comment:
Maybe unit test to see if is called with correct params ? 🤗

Reply:
Done, also implemented snapshots for filterable, custom filters, sortable, resizable and draggable